packages/api/src/router/post.ts

```diff
@@ -39,6 +40,7 @@ export const postRouter = createTRPCRouter({
         data: {
           title: input.title,
           contents: input.contents,
+          postType: "Discussion",
```

Comment:
is this field optional? don't think you need to add it have it for the update operation?

Reply:
lol yeah true removed that line